task9/index.js

```diff
@@ -0,0 +1,51 @@
+const obj = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+const div = document.getElementById("js-parent");
+
+const loading = () => {
+    const loadingPlace = document.createElement("div");
+    const gif = document.createElement('img');
+    loadingPlace.id = "loadingPlace";
+    gif.src = "loading-circle.gif";
+    div.appendChild(loadingPlace).appendChild(gif);
+};
+
+const removeLoading = () => {
+    document.getElementById("loadingPlace").remove();
+};
+
+const getData = () => {
+    return new Promise((resolve) => {
+        setTimeout(() => {
+            resolve(obj);
+        },2000);
+    });    
+};
+
+const objShow = async () => {
+    const value = await getData();
+    const ul = document.createElement('ul');
+    const fragment = document.createDocumentFragment();
+
+    removeLoading();
+    
+    value.forEach(item => {
```

Comment:
It seems that the process from here to line 48 can be separated as a function to generate list elements.

I think this will give you a better view of the code than you have now.